README.md

```diff
@@ -98,7 +98,7 @@ typeis.is(mediaType, ['html']) // false
 
 ## Examples
 
-### Example body parser
+#### Example body parser
```

Comment:
The level of the header was correct before; you're changing it to level 4 here but the parent is level 2.